util/util.go

```diff
@@ -474,7 +474,7 @@ func RetryOnErrorCondition(fn func() (interface{}, error), predicate func(error)
 		}
 		time.Sleep(APIRetryInterval + APIRetryJitterInterval*time.Duration(rand.Intn(5)))
 	}
-	return nil, fmt.Errorf("cannot finish API request due to too many error retries")
+	return nil, fmt.Errorf("cannot finish API request due to too many error retries since %v", err.Error())
```

Comment:
Why not use `errors.Wrapf()`?

Reply:
Yes, we can.

Reply:
Addressed.

Reply:
it seems not changed? still `fmt.Errorf`

Reply:
Okay. I revert the change here. Instead, I remove the retries for the failure of node/disk update because https://github.com/longhorn/longhorn-manager/pull/1327#issuecomment-1122346411.